### ISSUE TITLE
new openssh private key format

### ISSUE THREAD

**Issue by cypa** [S2-/sshenc.sh#7]
```
~λ openssl rsautl -decrypt -ssl -inkey .ssh/id_rsa
unable to load Private Key
139994616321280:error:0909006C:PEM routines:get_name:no start line:crypto/pem/pem_lib.c:745:Expecting: ANY PRIVATE KEY

```
https://github.com/jeroen/openssl/issues/66

**Comment by cypa** [comment]
`ssh-keygen -p -f id_rsa -m pem` converted the key to PEM, but decrypt do not work
```
~λ bash sshenc.sh -s id_rsa < 30-touchpad.conf.sshenc
Enter pass phrase for id_rsa:
RSA operation error
140402911679744:error:04072073:rsa routines:RSA_padding_check_SSLv23:sslv3 rollback attack:crypto/rsa/rsa_ssl.c:166:
140402911679744:error:04065072:rsa routines:rsa_ossl_private_decrypt:padding check failed:crypto/rsa/rsa_ossl.c:491:

```

**Comment by 5im-0n** [comment]
fixed in 845ad71fc43205147a814
